lib/windows/app/index.js

```diff
@@ -61,6 +61,18 @@ Module._load = function load(path) { // eslint-disable-line no-underscore-dangle
             return ReactDOM;
         case 'react-redux':
             return ReactRedux;
+        case 'nrfconnect/logger':
+            return api.logger;
+        case 'nrfconnect/pc-ble-driver-js':
+            return api.bleDriver;
```

Comment:
I think the reason why the `api` name was chosen in the first place in pc-ble-driver-js was to differentiate the api and the node bindings `driver`. The structure has since been modified so the `driver` is not exposed like before, so I think we could safely remove the `api` level and just expose the objects at top level.
I don't think it should be a problem to do this now. We would of course have to release with a new major version and modify tests and examples.